src/main/flight/pid.h

```diff
@@ -245,4 +250,4 @@ float calcHorizonLevelStrength(void);
 void dynLpfDTermUpdate(float throttle);
 void pidSetItermReset(bool enabled);
 float pidGetPreviousSetpoint(int axis);
-
+float pidGetFfBoostFactor();
```

Comment:
Your camel casing of `Ff` is inconsistent. `Ff` (i.e. only the first letter of a camel cased abbreviation is capitalised) is consistent with the pattern used in Betaflight.